main.tf

```diff
@@ -1,38 +1,42 @@
-module "user_label" {
-  source  = "cloudposse/label/null"
-  version = "0.25.0"
-  context = module.this.context
-
-  delimiter           = coalesce(module.this.context.delimiter, "_")
-  regex_replace_chars = coalesce(module.this.context.regex_replace_chars, "/[^_a-zA-Z0-9]/")
-  label_value_case    = coalesce(module.this.context.label_value_case, "upper")
+data "context_label" "this" {
+  delimiter  = local.context_template == null ? var.naming_scheme.delimiter : null
```

Comment:
do we need condition here? I think if template is set then the value of `delimiter` is irrelevant?

Reply:
Yes, we do need it - docs clearly say, that `delimiter` conficts` with `template`